module-2/backend/week_one.md

```diff
@@ -1,34 +1,80 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
+1.List the five common HTTP verbs and what the purpose is of each verb.
+  *Get-Is used to return data from the database. It is used to render a view.
+  *Post-Is used to create a database entry.
+  *Put-Is used to update an existing database entry.
+  *Patch-Is used to update specific attributes of a database entry.
+  *Delete-Is used to delete a database entry.
 2. What is Sinatra?
+  *A framework for ruby.

```

Comment:
It is a lightweight framework...that is a key difference between sinatra and rails